.github/workflows/dylint.yml

```
@@ -0,0 +1,95 @@
+            ${{ github.job }}-workspace-${{ runner.os }}-${{ hashFiles('rust-toolchain.toml') }}-
+
+      - name: Cache dylint lint library
+        uses: actions/cache@v5
+        timeout-minutes: 2
+        continue-on-error: true
+        with:
+          path: target/dylint
+          key: ${{ github.job }}-dylint-${{ runner.os }}-${{ hashFiles('rust-toolchain.toml') }}-${{ hashFiles('dylint.toml') }}
+
+      - name: Install Rust Toolchain
+        uses: ./.github/actions/rustup
```

actionlint (1.7.12)

[error] could not parse action metadata in "/home/jailuser/git/.github/actions/rustup": line 7: unexpected key "type" for definition of input "shared-key"

(action)